Formula/s/sidekick-cli.rb

```diff
@@ -0,0 +1,20 @@
+class SidekickCli < Formula
+  desc "Deploy applications to your VPS"
+  homepage "https://github.com/MightyMoud/sidekick"
+  url "https://github.com/MightyMoud/sidekick/archive/refs/tags/v0.5.3.tar.gz"
+  sha256 "11c6f23c67c122ec6e1e0fa4cd7119f7e044186abe810e08ea97708bb78e0a33"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+    bin.install bin/"sidekick-cli" => "sidekick"
+  end
+
+  test do
+    assert_match "With sidekick you can deploy any number of applications to a single VPS",
+                  shell_output(bin/"sidekick")
+    assert_match(/Sidekick config not found - Run sidekick init/, shell_output(bin/"sidekick compose", 1))
```

Comment:
```suggestion
    assert_match(/Sidekick config not found - Run sidekick init/, shell_output("#{bin}/sidekick compose", 1))
```
`bin/"sidekick compose"` is not a `Pathname`. (But `bin/"sidekick"` is.)